kie-server-parent/kie-server-controller/kie-server-controller-rest/src/main/java/org/kie/server/controller/rest/RestKieServerControllerImpl.java

```diff
@@ -58,11 +68,15 @@ public Response connectKieServer(@Context HttpHeaders headers,
         logger.debug("Server info {}", serverInfo);
         KieServerSetup serverSetup = connect(serverInfo);
 
-        logger.info("Server with id '{}' connected", id);
-        String response = marshal(contentType, serverSetup);
+        if (serverSetup.hasNoErrors()) {
+            logger.info("Server with id '{}' connected", id);
+        } else {
+            logger.info("Server with id '{}' failed to connect", id);
```

Comment:
Would it have sense to raise log severity to warning?

Reply:
make sense.